native/src/components/EastereggImage.tsx

```diff
@@ -49,6 +49,7 @@ class EastereggImage extends React.Component<PropsType, StateType> {
         })
       )
       .catch(e => {
+        // eslint-disable-next-line no-console
         console.error(e)
```

Comment:
Would it be possible to exclude `console.error` from this rule?

Reply:
@klinzo 
    'no-console': ['error', { allow: ['error', 'warn'] }],
this should do the trick